codewof/programming/content/en/valid-pin/question.md

```diff
@@ -0,0 +1,7 @@
+#Valid Pin
+
+Write a function `is_valid_pin(pin)` that takes a string `pin` as an argument and returns a boolean `True` if `pin` is a valid pin-number. Otherwise return `False`
```

Comment:
```suggestion
Write a function `is_valid_pin(pin)` that takes a string `pin` as an argument and returns a boolean `True` if `pin` is a valid pin-number. Otherwise return `False`.
```